src/main/resources/avro/bdg.avdl

```diff
@@ -381,7 +381,9 @@ record Variant {
 
   /**
    The zero-based, exclusive end position of this variant on the reference contig.
-   Calculated by start + referenceAllele.length().
+   If this variant is non-symbolic, this is calculated by start + referenceAllele.length().
+   If this variant is symbolic, this is populated from the VCF INFO reserved
+   key "END".
```

Comment:
Just to confirm: It appears to me that indeed the END from a reference/symbolic block record in a gVCF actually turn out to be correctly interpreted as  "zero-based" here because we are using it as the exclusive end position while  the gVCF interprets it as the inclusive END I think.   So - assuming that is correct, this code is fine as is, though we could add to the comment to make it even clearer lest anyone be concerned that END is correctly interpreted as zero-based exclusive in all cases.